app/modules/asset_groups/metadata.py

```diff
@@ -90,6 +91,31 @@ def _validate_sighting(self, sighting, file_dir, sighting_debug, encounter_debug
                 # Set ensures no duplicates
                 self.files.add(filename)
 
+        if 'idConfigs' in sighting:
+            id_config_fields = [
+                ('algorithms', list, True),
+                ('matchingSetDataOwners', str, True),
+                ('matchingSetRegions', list, False),
+            ]
+
+            num_configs = len(sighting['idConfigs'])
+            if num_configs > 1:
+                raise AssetGroupMetadataError(
+                    f'found multiple {num_configs} ID configs, only support one'
+                )
```

Comment:
Is this to prevent more than one config for now but  leave the door open to multiple configs post-MVP?

Reply:
correct

Reply:
You should also spot that once the restrictions that prevent multiple configs and algorithms are removed, the code should JustWork(TM) with them all